editor/src/clj/editor/workspace.clj

```diff
@@ -450,22 +457,25 @@ ordinary paths."
     (if (.exists dir)
       (delete-directory-recursive dir))))
 
-(def ^:private bin-zip-names
-  (into #{"/plugins/bin/common.zip"}
-        (comp
-          (mapcat (juxt #(.getPair ^Platform %) #(.getOs ^Platform %)))
-          (map #(str "/plugins/bin/" % ".zip")))
-        [Platform/X86_64Linux Platform/X86_64MacOS Platform/X86_64Win32]))
+(def ^:private plugin-zip-names
+  (let [platform (Platform/getHostPlatform)]
+    {"/plugins/common.zip" 0
+     (str "/plugins/" (.getOs platform) ".zip") 1
+     (str "/plugins/" (.getPair platform) ".zip") 2}))
+
+(defn- plugin-zip-priority [resource]
+  (plugin-zip-names (re-find #"/plugins/[^/]+\.zip" (resource/proj-path resource))))
 
-(defn- bin-zip? [resource]
+(defn- plugin-zip? [resource]
   (let [path (resource/proj-path resource)]
-    (boolean (some #(string/ends-with? path %) bin-zip-names))))
+    (some? (some #(string/ends-with? path %) (keys plugin-zip-names)))))
 
-(defn- unpack-bin-zip! [workspace resource]
+(defn- unpack-plugin-zip! [workspace resource]
   {:pre [(string/ends-with? (resource/proj-path resource) ".zip")]}
+  (unpack-resource! workspace resource)
   (let [proj-path (resource/proj-path resource)
         plugin-file (plugin-path workspace proj-path)
-        infix-path (subs proj-path 0 (- (count proj-path) 4))]
+        infix-path (resource/parent-proj-path proj-path)]
```

Comment:
This appears to no longer use the name of the zip archive. Is this correct?

Previously it just stripped away the `.zip` extension, but now it strips away the entire file name.

Reply:
Yes, it's correct. As described in the commit and PR description, we use the following unpack rule:
```
${ext}/plugins/${platform}.zip -> /build/plugins/${ext}/plugins/
```
as suggested by @JCash. That way we can pack both `lib/${platform}` and `bin/${platform}` resources in a single archive.